modules/ezcontent_node/modules/ezcontent_smart_article/src/Plugin/Field/FieldWidget/SmartTextareaWithSummaryWidget.php

```diff
@@ -24,29 +24,46 @@ class SmartTextareaWithSummaryWidget extends TextareaWithSummaryWidget {
    */
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     $element = parent::formElement($items, $delta, $element, $form, $form_state);
-    $element['summary_type'] = [
+    $element['summary']['#prefix'] = '<div class="text-summary-wrapper">';
+    $element['summary']['#attributes']['class'] = ['text-summary'];
+    $element['summary']['#attributes']['style'] = 'display: none;';
```

Comment:
Drupal core provides a class `hidden` to hide things. Please see if you can reuse.
refer - https://git.drupalcode.org/project/drupal/blob/8.8.x/core/modules/system/css/components/hidden.module.css